logic/resolvable/Concludable.java

```diff
@@ -268,67 +267,61 @@ public Relation(final RelationConstraint constraint, @Nullable IsaConstraint isa
                 } else return Iterators.empty();
             }
 
-            // TODO this will work for now, but we should rewrite using role player `repetition`
             List<RolePlayer> conjRolePlayers = list(constraint().players());
```

Comment:
Since we are recursing by "peeling" off the Conjunction Role Players, we still them in a `List` structure.